recipes/segment-lidar/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "segment-lidar" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/segment-lidar-{{ version }}.tar.gz
+  sha256: a5a60c2823c7c24d510ceccce93f365cb2585e9e75caefb1c59c5cbebdaff293
+
+build:
+  skip: True
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - pip
+    - pycocotools ==2.0.6
+    - click ==8.1.3
+    - opencv ==4.7.0.72
+    - numpy ==1.24.3
+    - supervision ==0.8.0
+    - segment-geospatial ==0.8.1
+    - rasterio ==1.3.7
+    - laspy ==2.4.1
+    - laszip ==0.2.3
```

These are super tight and very prone to breaking. I would report upstream and ask for a more relaxed, min ver only setup.

```suggestion
    - pycocotools ==2.0.6
    - click ==8.1.3
    - opencv ==4.7.0.72
    - numpy ==1.24.3
    - supervision ==0.8.0
    - segment-geospatial ==0.8.1
    - rasterio ==1.3.7
    - laspy ==2.4.1
    - laszip ==0.2.3
```

---

Hello @ocefpaf, thanks for reviewing my PR. This is the first time I'm trying to publish a package on `conda-forge`, so I don't really get the problem with these requirements, is it because of versions?

---

Helle @ocefpaf, thanks for reviewing my PR. I'll make them minimum requirements if that works?

---

I believe these are correct with upstream, right? I would leave them as is here and report that upstream first.